mars/api.py

```diff
@@ -148,6 +148,14 @@ def get_graph_state(self, session_id, graph_key):
         return state
 
     def fetch_data(self, session_id, graph_key, tileable_key, index_obj=None, compressions=None):
+        if graph_key is None or graph_key == 'None':
```

Comment:
The 'None' str looks pretty weird here, could the `graph_key` be None only?

Reply:
The `graph_key` comes from the URL, where `None` is parsed as `'None'`. It is wired, and I could parse it as `None` in the web api handler....